sql-ballerina/types.bal

```diff
@@ -325,9 +325,9 @@ public distinct class TimestampValue {
 # + value - Value of parameter passed into the SQL statement
 public distinct class ArrayValue {
     *TypedValue;
-    public string[]|int[]|boolean[]|float[]|decimal[]|byte[][]? value;
+    public Value[]|anydata[]? value;
```

Comment:
Do you need both Value[] and anydata[] here?

Reply:
One suggestion is to Change it to `TypedValue[]|anydata[]`

Reply:
Is anydata needed to have combined array? i.e array with both int and string mixed in them?

Reply:
No. Its need to accept record Arrays.
Such as `Point arrays` in PostgreSQL.